File: cpp/Ram.cpp
#include <iostream>
#include <string>

using namespace std;

class Ram
{
private:
    int CapacityRam;
    int PriceRam;

public:
    Ram(int Capacity, int Price)
    {
        this->CapacityRam = Capacity;
        this->PriceRam = Price;
    }

    Ram() {}

    void setCapacityRam(int Capacity)
    {
        this->CapacityRam = Capacity;
    }

    int getCapacityRam()
    {
        return this->CapacityRam;
    }

    void setPriceRam(int Price)
    {
        this->PriceRam = Price;
    }

    int getPriceRam()
    {
        return this->PriceRam;
    }

    void print()
    {
        cout << "Ram :" << endl;
        cout << endl;
        cout << "Capacity (Gb) -> " << this->getCapacityRam() << endl;
        cout << "Price (Rp) ->" << this->getPriceRam() << endl;
    }

    ~Ram() {}
};

File: python/Ram.py
class Ram:

    __CapacityRam = 0
    __PriceRam = 0

    def __init__(self, Capacity = 0, Price=0):

        self.__CapacityRam = Capacity
        self.__PriceRam = Price


    def setCapacityRam(self, Capacity):
        self.__CapacityRam = Capacity

    def getCapacityRam(self):
        return self.__CapacityRam
        
    def setPriceRam(self, Price):
        self.__PriceRam = Price

    def getPriceRam(self):
        return self.__PriceRam

    def print(self):
        print("Ram:")
        print("Capacity (Gb) : " + str(self.getCapacityRam()))
        print("Price (Rp) :" + str(self.getPriceRam()))